src/geometries.jl

```diff
@@ -146,11 +146,11 @@ parametric dimension of the polytope. For example, a segment is a 1-face, a tria
 """
 abstract type Face{N,Dim,T} <: Polytope{N,Dim,T} end
 
-(::Type{F})(vertices::Vararg{P}) where {F<:Face,P<:Point} = F(SVector(vertices))
-(::Type{F})(vertices::AbstractVector{TP}) where {F<:Face,TP<:Tuple} = F(Point.(vertices))
-(::Type{F})(vertices::Vararg{TP}) where {F<:Face,TP<:Tuple} = F(collect(vertices))
+(::Type{F})(vertices::Vararg{TP}) where {F<:Polytope,TP<:Tuple} = F(collect(vertices))
+(::Type{F})(vertices::AbstractVector{TP}) where {F<:Polytope,TP<:Tuple} = F(Point.(vertices))
+(::Type{F})(vertices::Vararg{P}) where {F<:Polytope,P<:Point} = F(SVector(vertices))
 
-==(f1::Face, f2::Face) = f1.vertices == f2.vertices
+==(p1::Polytope, p2::Polytope) = p1.vertices == p2.vertices
```

Comment:
Actually, I am not sure about this one. We can define this as a default, which will be fine for all common implementations _if the ordering is the same_, but essentially a polytope is not just 0-faces, it's all k-faces. So, testing equality for vertices should just `vertices(p1) == vertices(p2)`, but otherwise the polytope structure may not be the same. Actually it could be interesting to define a full polytope structure, and it deserves its own issue.

Reply:
It would be nice to learn more about the structure you mentioned. I will comment on the other issue you opened :+1: